consoleme/lib/policies.py

```diff
@@ -715,8 +715,20 @@ async def get_resource_name_for_arn(arn: str) -> str:
     return resource_name
 
 
+async def get_resource_sub_type_for_arn(arn: str) -> str:
+    resource_name = arn.split(":")[5]
+    if "/" in resource_name:
+        return resource_name.split("/")[0]
+    return ""
+
+
 async def get_url_for_resource(
```

Comment:
It'd be cool to use this functionality within ConsoleMe as well! Future feature?

IE:  www.consoleme.url/resource/arn:aws:s3:::bucket_name

Reply:
Ooooh I like this idea a lot; that would be super cool!